### PR TITLE
Translator App update

### PR DESCRIPTION
**Pull request by martinxu9**

### CODE REVIEW

File: translator/translator/translator.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+            text_box(message.original_text),
+            down_arrow(),
+            text_box(message.text),
+            rx.hstack(
+                rx.text(message.to_lang),
+                rx.text("·"),
+                rx.text(message.created_at),
+                spacing="2",
+                font_size="0.8rem",
+                color="var(--gray-11)",
```
Comment (masenf) [inline translator/translator/translator.py:105]: we should use `rx.color("gray", 11)` here instead

same with other instances of explicit `var(...)` usage

File: translator/translator/translator.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+def translation_form() -> rx.Component:
+    """This is a form that contains:
+    - Text input: what to translate
+    - Select: the destination language
+    - Output: the translation
+    - Submit button: to post the text input as part of the past translations view
+    """
+    return rx.form(
+        rx.flex(
+            rx.input.input(
```
Comment (masenf) [inline translator/translator/translator.py:155]:
```suggestion
            rx.input(
```

rx.input is just an alias for the textfield input now, no need for the double input

**Review by masenf — CHANGES_REQUESTED** [review]

**Comment by martinxu9** [comment]
@masenf updated

File: translator/translator/translator.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+                placeholder="Select a language",
+                items=LANGUAGE_NAMES,
+                value=State.lang,
+                on_change=State.set_lang,
+                margin_top="1rem",
+                size="3",
+                width="100%",
+            ),
+            output(),
+            rx.form.submit(
```
Comment (picklelo) [inline translator/translator/translator.py:174]: Should we use `rx.form.submit` or use `rx.button(type="submit")` which is the current way?

**Review by picklelo — APPROVED** [review]

File: translator/translator/translator.py (hunk 3 above)
Comment (masenf) [reply translator/translator/translator.py]: actually `rx.button` defaults to a submit button now; since in html that is the default.

Comment (martinxu9) [reply translator/translator/translator.py]: @picklelo think you have a question about that

Comment (picklelo) [reply translator/translator/translator.py]: Do all buttons default bo submit? What if there's a use case where they want another button in their form?

Comment (picklelo) [reply translator/translator/translator.py]: Okay seems this is the default behavior now, we can keep it

**Review by picklelo — APPROVED** [review]

**Pull request merged**